lib/SPIRV/libSPIRV/SPIRVOpCodeEnum.h

```diff
@@ -429,7 +431,7 @@ _SPIRV_OP(SubgroupAvcSicGetIpeChromaModeINTEL, 5813)
 _SPIRV_OP(SubgroupAvcSicGetPackedSkcLumaCountThresholdINTEL, 5814)
 _SPIRV_OP(SubgroupAvcSicGetPackedSkcLumaSumThresholdINTEL, 5815)
 _SPIRV_OP(SubgroupAvcSicGetInterRawSadsINTEL, 5816)
+_SPIRV_OP(LoopControlINTEL, 5887)
```

Comment:
I think it is better to remove this unrelated change

Reply:
I just fixed the order of instructions. Do you want to make it in a separate commit?